### PR TITLE
Feature to use kafka to send span

### PR DESCRIPTION
**Pull request by rakeshkaundilya**
1. Added two variables in Zipkin Factory: bootstrapServers, topic : both required for kafka producer.
2. Created span Collector using brave kafka span collector.

Signed-off-by: Rakesh Kumar <rakeshkumar4294@gmail.com>

### CODE REVIEW

File: zipkin-core/pom.xml [hunk 1]
```diff
@@ -26,5 +26,10 @@
             <artifactId>brave-spancollector-http</artifactId>
             <version>${brave.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.github.kristofa</groupId>
+            <artifactId>brave-spancollector-kafka</artifactId>
+            <version>3.7.0</version>
```
Comment (adriancole) [inline zipkin-core/pom.xml]: shouldn't this use the same version variable? `${brave.version}`

File: zipkin-core/src/main/java/com/smoketurner/dropwizard/zipkin/ZipkinFactory.java [hunk 2]
```diff
@@ -48,6 +49,9 @@
             .getLogger(ZipkinFactory.class);
     private static final String DEFAULT_ZIPKIN_SCRIBE = "127.0.0.1:9140";
 
+    private String bootstrapServers;
```
Comment (adriancole) [inline zipkin-core/src/main/java/com/smoketurner/dropwizard/zipkin/ZipkinFactory.java]: I don't know how the configuration ends up looking.. would these be obviously scoped to kafka? (remembering there are multiple transports defined here)

Comment (rakeshkaundilya) [inline zipkin-core/src/main/java/com/smoketurner/dropwizard/zipkin/ZipkinFactory.java]: In my local host I can receive message at Kafka consumer console.

File: zipkin-core/src/main/java/com/smoketurner/dropwizard/zipkin/ZipkinFactory.java [hunk 3]
```diff
@@ -48,6 +49,9 @@
...
+    private String topic;
```
Comment (jplock) [inline zipkin-core/src/main/java/com/smoketurner/dropwizard/zipkin/ZipkinFactory.java]: Is there a default topic (maybe `zipkin`) that we can provide?

Comment (adriancole) [inline zipkin-core/src/main/java/com/smoketurner/dropwizard/zipkin/ZipkinFactory.java]: yep the default is zipkin

**Comment by jplock** [comment]
I'll merge this in an clean it up a bit.  Thanks for the contribution!

**Pull request merged**